### DISCUSSION TITLE
Sample Print SCP

### DISCUSSION THREAD

**Discussion by jeffersonarar** [fo-dicom/fo-dicom#1380] [Q&A]
Boa tarde meus caros!

Estamos testanto o Sample Print SCP, selecionamos a impressora passamos o nome corretamente, porém da o seguinte erro.
Message = "Cannot cast to 'Image'; type must be assignable from 'Byte[]'
Testamos na versão 4.0.0 ok
Apenas na ultima versão  5.0.2 que não funcionou.

**Comment by gofal** [comment]
google translate:

We are testing the Sample Print SCP, we selected the printer we passed the name correctly, but it gives the following error.
Message = "Cannot cast to 'Image'; type must be assignable from 'Byte[]'
We tested in version 4.0.0 ok
Only in the last version 5.0.2 it didn't work.

**Comment by gofal** [comment]
you have to reference the fo-dicom.imaging.desktop nuget package and register the WinformsImageManager. See https://fo-dicom.github.io/dev/v5/usage/image_rendering.html
